tools/crm_resource.c

```diff
@@ -24,8 +24,40 @@
 #include <crm/stonith-ng.h>
 #include <crm/common/ipc_controld.h>
 
+struct {
+    bool clear_expired;
+    int find_flags;             /* Flags to use when searching for resource */
+    char *host_uname;
+    char *interval_spec;
+    char *operation;
+    GHashTable *override_params;
+    char *prop_id;
+    char *prop_name;
+    char *prop_set;
+    char *prop_value;
+    bool recursive;
+    bool require_crmd;          /* whether command requires controller connection */
+    bool require_dataset;       /* whether command requires populated dataset instance */
+    bool require_resource;      /* whether command requires that resource be specified */
+    char rsc_cmd;
+    char *rsc_id;
+    char *rsc_long_cmd;
+    char *rsc_type;
+    bool scope_master;
```

Comment:
Note that scope_master is used a bunch of places in crm_resource.h, crm_resource_ban.c, and crm_resource_runtime.c.  If you'd like I can also change those.  I'd do it in a separate commit since there's nothing else changing in those files in the first commit here.

Reply:
Might as well, I expect we'll be doing that in the next 6-9 months anyway

Reply:
Fixed.